cmd/opentelemetry/app/exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -127,3 +126,15 @@ func (f Factory) CreateLogsExporter(
 ) (component.LogsExporter, error) {
 	return f.Wrapped.CreateLogsExporter(ctx, params, cfg)
 }
+
+// mustOtelEncodingForJaegerEncoding translates a jaeger encoding to a otel encoding
+func mustOtelEncodingForJaegerEncoding(jaegerEncoding string) string {
+	switch jaegerEncoding {
+	case kafka.EncodingProto:
```

Comment:
no otlp proto?

Reply:
My original thought is to add `otlp-proto` just for Kafka receiver, not Kafka exporter. But I'm glad to make Kafka exporter support that.